### PR TITLE
smalles fixes for beta-client

### PR DESCRIPTION
**Pull request by leonardlin**
# Description
* Fix compile error 'typo' in beta client: 'request' -> 'requests' 
* Add documentation how to use beta-client
* Updated main README.md about broken upload-file in old client.

Fixes # (issue)
#13 #16 #50

### CODE REVIEW

**Comment by david-engelmann** [comment]
This pull request (https://github.com/doccano/doccano-client/pull/71) would cover [this open pull request](https://github.com/doccano/doccano-client/pull/68) as well.

File: doccano_api_client/beta/tests/test_full_integration_tests.py [hunk 1]
```diff
@@ -5,7 +5,9 @@
 from ..client import DoccanoClient
 from ..models import Example, Label, Project
 
-DOCCANO_ENDPOINT = "http://0.0.0.0"  # TODO: Make this a pytest parameter
+DOCCANO_ENDPOINT = "http://localhost:8000"  # TODO: Make this a pytest parameter
+DOCCANO_USER = "admin"
+DOCCANO_PASS = "password"
```
Comment (ljades) [inline doccano_api_client/beta/tests/test_full_integration_tests.py:10]: Can you edit the TODO to make all of these pytest params or ENV variables in the future (actually, that might actually be even better for all three)?

**Review by ljades — CHANGES_REQUESTED** [review]
Thanks for handling this!

File: doccano_api_client/beta/tests/test_full_integration_tests.py (hunk 1 above)
Comment (ljades) [reply doccano_api_client/beta/tests/test_full_integration_tests.py:10]: @leonardlin bumping on this. When we've knocked it out, I'd love to see your PR on the 1.6/1.7 compatibility bump you mentioned in #72 too!

**Comment by ljades** [comment]
Good to go after black linter is resolved!

**Review by ljades — APPROVED** [review]

**Pull request merged**